src/components/ClusterOverview/Inventory/Inventory.js

```diff
@@ -15,16 +15,18 @@ import {
   mapPvcsToProps,
   mapVmsToProps,
   mapHostsToProps,
+  mapDiskStatsToProps,
 } from '../../Dashboard/Inventory/utils';
 import { InventoryRow } from '../../Dashboard/Inventory/InventoryRow';
 
-const InventoryBody = ({ nodes, pods, vms, vmis, pvcs, migrations, hosts }) => (
+const InventoryBody = ({ nodes, pods, vms, vmis, pvcs, migrations, hosts, diskStats }) => (
```

Comment:
where are these `diskStats` coming from ? Is there any PR for `web-ui` ? I also wonder how is it that the build is passing but the prop is missing in `propTypes` :(

Reply:
It's coming from prometheus, the web-ui PR is coming soon. It's very similar to the code in the storage dashboard.

Reply:
PR is up https://github.com/kubevirt/web-ui/pull/294